talks/README.md

```diff
@@ -2,3 +2,9 @@ The talks folder can contain any materials used for meetups at the Bay Area WTD
 
 This README is the place to store links to your recorded WTD events, if they are stored on YouTube. 
 
+**talks - Table of Contents** 
+
+| Date     | Loc | Topic                    | Filename <presenter_topic_name> | 
```

Comment:
Suggest clarifying the expected format in the Date column; e.g., "Date (yyyy/mm/dd)".